composer.json

```diff
@@ -4,12 +4,13 @@
   "type": "drupal-module",
   "license": "GPL-2.0-or-later",
   "require": {
-    "drush/drush": "^10.3",
+    "drush/drush": "^11",
     "drupal/purge": "^3.4",
+    "drupal/section_purge": "dev-3307199-drupal-10-compatibility",
```

Comment:
This is using a fork version because the name change is still in merge request for this module. Once it is tested with this fork version, then only we can merge that PR in the contrib module and the local fork version reference will be removed.